### PR TITLE
Phase 2 tracker: add T28 and T29 (geometries with bricked pixels for IT sensor design studies)

### PR DESCRIPTION
**Pull request by adewit**
#### PR description:

This PR adds two new phase2 tracker layouts proposed in the IT sensors performance task force and correspond to evolutions of T27 (geometry with bricked pixels). More information [[in these slides](https://indico.cern.ch/event/1072175/contributions/4648635/attachments/2362758/4033767/em20211210.pdf])] 

In particular this adds:
- T28, a tracker based on T25, with 25x100 mum2 3D pixels in IT TBPX layer 1, bricked pixels in the central rod of TBPX layers 2 and 3, in the central 3 rods of TBPX layer 4, in TEPX, and in disks 5-8 of TFPX. The other pixels are planar 25x100 mum2.

- T29, a tracker based on T25, with 25x100 mum2 3D pixels in IT TBPX layer 1, bricked pixels in the central rod of TBPX layers 2 and 3, in the central 3 rods of TBPX layer 4. 50x50 mum2 planar pixels are used in TFPX and TEPX. 

There are no changes in the OT wrt T25.

T28 is included in geometry scenario 2026D89 and T29 in scenario 2026D90

**important**: we use process modifiers to account for the presence of bricked pixels, but no square pixel process modifiers are applied (even in T29). The reason for this is that with the square pixel process modifier the genErrors are used when running the CPE without track angles, but we do not have genErrors for bricked pixels.

@emiglior @tsusa @connorpa fyi

#### PR validation:

- checked XMLs make sense wrt tracker design
- checked that the workflows with scenarios D89 and D90 run smoothly with no warnings.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport

### CODE REVIEW

**Comment by mmusich** [comment]
test parameters:
   * workflows = 39834.0, 40234.0

**Comment by mmusich** [comment]
@adewit 

> important: we use process modifiers to account for the presence of bricked pixels, but no square pixel process modifiers are applied (even in T29).

I guess you do that because otherwise you would fall in the case in which genErrors are used when the CPE is called w/o track angles and there are none for bricked pixels.
https://github.com/cms-sw/cmssw/blob/a4bb3edbab1a04064864021ee1a4096e53662c6a/RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py#L54-L56

If that's the case might be worth specifying in the PR description.

**Comment by adewit** [comment]
> @adewit
> 
> > important: we use process modifiers to account for the presence of bricked pixels, but no square pixel process modifiers are applied (even in T29).
> 
> I guess you do that because otherwise you would fall in the case in which genErrors are used when the CPE is called w/o track angles and there are none for bricked pixels.
> 
> https://github.com/cms-sw/cmssw/blob/a4bb3edbab1a04064864021ee1a4096e53662c6a/RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py#L54-L56
> 
> If that's the case might be worth specifying in the PR description.

Yes, that is the reason. Have updated the PR description to be more specific about this.

**Comment by mmusich** [comment]
please test

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
urgent

**Comment by srimanob** [comment]
Kindly ping @cms-sw/pdmv-l2, could you please consider?
Thanks.

**Comment by kskovpen** [comment]
+1

**Comment by perrotta** [comment]
please test
(just to refresh the one week old tests, while several geometry related changes had been merged in master since then)

**Comment by bsunanda** [comment]
@perrotta @qliphy Can this PR be integrated urgently?

File: Configuration/PyReleaseValidation/python/relval_2026.py [hunk 1]
```diff
@@ -40,6 +40,8 @@
 numWFIB.extend([38634.0]) #2026D86
 numWFIB.extend([39034.0]) #2026D87
 numWFIB.extend([39434.0]) #2026D88
+numWFIB.extend([39834.0]) #2026D89
+numWFIB.extend([40234.0]) #2026D89
```
Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_2026.py:44]:
```suggestion
numWFIB.extend([40234.0]) #2026D90
```

**Comment by perrotta** [comment]
+operations

**Comment by perrotta** [comment]
merge
(Fixed the geometry scenario in a comment field: no need to re-test)

**Pull request merged**